package.json

```diff
@@ -50,7 +49,9 @@
   },
   "devDependencies": {
     "eslint": "^6.6.0",
+    "eslint-plugin-prettier": "^3.2.0",
```

Comment:
I know the exact difference between `devDependencies` and `dependencies`. Can you explain the why this was move to `devDependencies`

Reply:
Prettier and eslint aren't used in production, so that's why dev dependencies. Build time will be shorter, less time for netlify to install packages.